### ISSUE TITLE
[Catalog] Publish WordPress and MySQL on Kubernetes

### ISSUE THREAD

**Issue by theBeginner86** [meshery/meshery#9284]
### Current Behavior

Publish [WordPress and MySQL on Kubernetes](https://github.com/kubernetes/examples/tree/master/mysql-wordpress-pd#example-wordpress-and-mysql-on-kubernetes) to [Meshery Catalog](https://github.com/kubernetes/examples/tree/master/mysql-wordpress-pd#example-wordpress-and-mysql-on-kubernetes).

### Mockups

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by ShatilKhan** [comment]
@theBeginner86 I'd like to take this one
I'll try to import this design from Kubernetes example : https://github.com/kubernetes/examples/tree/master/mysql-wordpress-pd

**Comment by theBeginner86** [comment]
@ShatilKhan 
How is this coming along? Any early progress to report? Any issues you are facing, by chance?

To speed up the process, you may want to use [Playground](https://play.meshery.io).

**Comment by ShatilKhan** [comment]
Yeah I'm working on it
Thanks for the playground suggestion
I'll share updates soon

**Comment by ShatilKhan** [comment]
@Raj100679 
Would be great if you can take up another issue
I'm gonna be making my PR today

**Comment by Raj100679** [comment]
Sure @ShatilKhan , Can you guide me how are you doing this?

**Comment by ShatilKhan** [comment]
> Sure @ShatilKhan , Can you guide me how are you doing this?

Yeah all the examples that need to be implemented can be found here
https://github.com/kubernetes/examples
There’s similar issues like this one you can check out
They all require the examples from the provided link

**Comment by sandramsc** [comment]
This issue has been open for some time with no recent activity, unassigning to open it up for new contributors to give it a go.

**Comment by dshunter107** [comment]
Hello. May I give this a go @sandramsc

**Comment by sandramsc** [comment]
Hi @dshunter107 I have assigned this to you.

**Comment by dshunter107** [comment]
Hello @sandramsc 
I submitted the design to the catalog in the playground. However, when I try to put in the details in the "Description" and "Caveats and Considerations," I get an error message, and it does not save. As a result, I can only put the design in without any explanation for the Meshery Catalog. Here is what I would put in each section:

Description 
 - This model shows you how to deploy a WordPress site and a MySQL database. Both WordPress and MySQL use PersistentVolumes and PersistentVolumeClaims to store data.

Caveats and Considerations 
- This deployment is not suitable for production use cases, as it uses single instance WordPress and MySQL Pods. Consider using WordPress Helm Chart to deploy WordPress in production.

I am not sure what I should do because of the error message. Any advise would be welcome.

**Comment by dshunter107** [comment]
@theBeginner86 ^^

**Comment by sandramsc** [comment]
CC: @sudhanshutech

**Comment by DEVSAINI29** [comment]
Can you assign this issue to me? I'm ready to start!

**Comment by Bharath314** [comment]
Looks like this issue might have been [fixed](https://meshery.io/catalog/deployment/wordpress-and-mysql-with-persistent-volume-on-kubernetes-2df29b2b-7e5e-4d8b-a5b3-3dea569f4d25.html)?

**Comment by ty239** [comment]
Can you assign this issue to me

**Comment by lucasbryant1222-png** [comment]
As Bharath mentioned, it appears this issue regarding the WordPress and
MySQL publication on Kubernetes may have already been resolved. You can
check the status here:
https://meshery.io/catalog/deployment/wordpress-and-mysql-with-persistent-volume-on-kubernetes-2df29b2b-7e5e-4d8b-a5b3-3dea569f4d25.html

**Comment by redfordwesley320-arch** [comment]
Thank you for the update. Since the issue regarding the WordPress and MySQL
publication on Kubernetes appears to be resolved as previously noted by
Bharath, I will go ahead and close this issue.

**Comment by ishwar170695** [comment]
Closing this issue as the requested design has been published to the Meshery Catalog: https://meshery.io/catalog/deployment/wordpress-and-mysql-with-persistent-volume-on-kubernetes-2df29b2b-7e5e-4d8b-a5b3-3dea569f4d25.html

If there is any remaining work not covered by the published design, feel free to reopen the issue.
